omrmakefiles/rules.aix.mk

```diff
@@ -57,28 +57,24 @@ else
 endif
 GLOBAL_CPPFLAGS+=-D_XOPEN_SOURCE_EXTENDED=1 -D_ALL_SOURCE -DRS6000 -DAIXPPC -D_LARGE_FILES
 
-ifeq (,$(findstring xlclang,$(notdir $(CC))))
-  ifeq ($(OMR_ENV_OPENXL),1)
-    # openxl options
-    GLOBAL_CFLAGS+=-q mbcs -std=c89 -qinfo=pro
-    GLOBAL_CFLAGS+=-std=c++11
-  else
-    # xlc options
-    GLOBAL_CFLAGS+=-q mbcs -qlanglvl=extended -qinfo=pro
-  endif
+ifeq (openxl,$(OMR_TOOLCHAIN))
+  # openxl options
+  GLOBAL_CFLAGS+=-q mbcs -std=c89 -qinfo=pro
+  GLOBAL_CFLAGS+=-std=c++11
+else ifneq (,$(findstring xlclang,$(notdir $(CC))))
+  # xlc options
+  GLOBAL_CFLAGS+=-q mbcs -qlanglvl=extended -qinfo=pro
 else
   # xlclang options
   GLOBAL_CFLAGS+=-qlanglvl=extended0x -qxlcompatmacros
```

Comment:
When inverting the code, the bodies if and else branches must be swapped as well.